react/react1/week2/to-do-list/src/App.css

```diff
@@ -0,0 +1,14 @@
+.Container {
+  margin: 20px;
+}
+
+.listItems {
+  font-size: 16px;
+  line-height: 2em;
+  list-style-type: none;
+}
```

Comment:
Some classes starts with lower case, some with uppercase. Usually the convention is to use lowercase everywhere.

I'd rename this one to `.container` instead of `.Container`